### DISCUSSION TITLE
Apply custom ssl certificate to multiple hosts

### DISCUSSION THREAD

**Discussion by liviom** [NginxProxyManager/nginx-proxy-manager#3160] [Q&A]
Hello to everyone 
There is the way to apply a custom ssl certificate to all hosts?
Actually I have more than 80 host and apply the same ssl certificate to all is a long operation, I need to edit, select the ssl and apply it to all, one by one

Thanks in advance for your support

**Comment by lastsamurai26** [comment]
I have the same problem, I looked up the path of the certificate and simply replaced the contents of the certificate and then simply restarted the nginx. There is no other possibility, unfortunately, re-uploading a certificate does not work.

I have already opened a request 
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3153
